<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWindow</class>
 <widget class="QDialog" name="AdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Бюро трудоустройства</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>0</y>
     <width>500</width>
     <height>625</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>625</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>625</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Добавить работодателя</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>72</x>
       <y>9</y>
       <width>350</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>351</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Логин</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="NameEdit">
     <property name="geometry">
      <rect>
       <x>72</x>
       <y>184</y>
       <width>350</width>
       <height>35</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="AdressEdit">
     <property name="geometry">
      <rect>
       <x>72</x>
       <y>328</y>
       <width>350</width>
       <height>35</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="LoginEdit">
     <property name="geometry">
      <rect>
       <x>72</x>
       <y>40</y>
       <width>350</width>
       <height>35</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>72</x>
       <y>369</y>
       <width>350</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>351</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Телефон</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="PhoneEdit">
     <property name="geometry">
      <rect>
       <x>72</x>
       <y>400</y>
       <width>350</width>
       <height>35</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="PasswordEdit">
     <property name="geometry">
      <rect>
       <x>72</x>
       <y>112</y>
       <width>350</width>
       <height>35</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="CreateButton">
     <property name="geometry">
      <rect>
       <x>159</x>
       <y>477</y>
       <width>175</width>
       <height>35</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>175</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>175</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Создать</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>72</x>
       <y>225</y>
       <width>350</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>351</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Вид деятельности</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>72</x>
       <y>297</y>
       <width>350</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>351</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Адрес</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>72</x>
       <y>81</y>
       <width>350</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>351</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Пароль</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>72</x>
       <y>153</y>
       <width>350</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>351</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Название</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ActivityTypeEdit">
     <property name="geometry">
      <rect>
       <x>72</x>
       <y>256</y>
       <width>350</width>
       <height>35</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Удалить пользователя</string>
    </attribute>
    <widget class="QLineEdit" name="LoginEdit_2">
     <property name="geometry">
      <rect>
       <x>72</x>
       <y>184</y>
       <width>350</width>
       <height>35</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>72</x>
       <y>153</y>
       <width>350</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>351</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Логин</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CreateButton_2">
     <property name="geometry">
      <rect>
       <x>159</x>
       <y>477</y>
       <width>175</width>
       <height>35</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>175</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>175</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
